src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -29,6 +29,7 @@
     <ProjectCapability Include="ManagedLang"/> <!-- Temporary: See https://github.com/dotnet/roslyn-project-system/issues/47 -->
     <ProjectCapability Include="HandlesOwnReload"/>
     <ProjectCapability Include="UseFileGlobs"/>
+    <ProjectCapability Include="ManagedEditAndContinue"/>
```

Comment:
Why not just call this capability "EditAndContinue"? We don't use the Managed prefix for capabilities anywhere else.